common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -1135,8 +1135,15 @@ func (d *cassandraPersistence) CreateWorkflowExecution(request *p.CreateWorkflow
 					}
 				}
 
-				msg := fmt.Sprintf("Workflow execution creation condition failed. WorkflowId: %v, columns: (%v)",
-					request.Execution.GetWorkflowId(), strings.Join(columns, ","))
+				if prevRunID := previous["current_run_id"].(gocql.UUID).String(); prevRunID != request.Execution.GetRunId() {
```

Comment:
will this throw nil pointer error in case previous does not have "current_run_id"?

Reply:
There should be no case `current_run_id` is not in previous. But for resiliency, I can add a check which might be dead code.